app/utils/post-data.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+const postData = async (formData) => {
+  const applicationMutation = `mutation CreateApplication($formData: JSON = "formData") {
+  createApplication(input: {application: {formData: $formData}}) {
+    clientMutationId
+  }
+}`;
+
+  try {
+    await axios({
+      method: "POST",
+      url: 'http://localhost:3000/graphql',
```

Comment:
Next step: let's ensure we're passing this from an environment variable through next so it's available in both client and server side next renders and isn't statically compiled into the bundle during next build.